packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -61,6 +61,7 @@
     "pictureInPictureTileAriaLabel": "Video Feeds. Click to return to call screen.",
     "removeMenuLabel": "Remove",
     "copyInviteLinkButtonLabel": "Copy invite link",
+    "dismissSidePaneButton": "Close",
```

Comment:
Should this maybe say `close side pane`? just saying close the user might think its closes or leaves the app?

Reply:
Sounds like that would be the right wording. But A11y recommended to write "Close". 
https://skype.visualstudio.com/SPOOL/_workitems/edit/2790819/